run.sh

```diff
@@ -102,9 +102,10 @@ es_node_start=" --network devnet \
   --l1.beacon-based-time 1705546368 \
   --l1.beacon-based-slot 7419864 \
   --download.thread 32 \
+  --p2p.listen.udp 30305 \
```

Comment:
We may also need to change run-rpc.sh, run-docker.sh, docker-compose.yaml, and Dockerfile